packages/components/pie-cookie-banner/test/component/pie-cookie-banner.spec.ts

```diff
@@ -251,9 +215,9 @@ test.describe('PieCookieBanner - Component tests', () => {
     });
 
     test.describe('`locale` prop', () => {
-        test('should render text in the default (English) language when the locale is not set', async ({ mount }) => {
+        test('should render text in the default (English) language when the locale is not set', async () => {
             // Arrange
-            await mount(PieCookieBanner, {});
+            await pieCookieBannerComponent.load({ defaultPreferences: null });
```

Comment:
Was it necessary to add this?
```suggestion
            await pieCookieBannerComponent.load();
```

Reply:
Actually maybe just make an object that contains all of the preferences but with them explicitly set to false, then you can use that in all these preference tests